std/sumtype.d

```diff
@@ -2615,3 +2627,10 @@ private void destroyIfOwner(T)(ref T value)
         destroy(value);
     }
 }
+
+// https://github.com/dlang/phobos/issues/10563
+// Do not waste space for tag if sumtype has only single type
+unittest
+{
+    static assert(SumType!int.sizeof == 4);
+}
```

Comment:
Please put this `unittest` block below the other unit tests for `SumType` (above `private enum bool isSumTypeInstance`), not at the end of the file.